src/components/Pages/ManageWiki/Features/ItemsTable.vue

```diff
@@ -191,6 +197,8 @@ export default {
 
     deleteItemConfirm () {
       this.wikibaseItems.splice(this.editedIndex, 1)
+      // TODO: this way of deleting an item pair is silly. how to do computed setters for arrays?!
```

Comment:
I think the answer to this smart comment that I wrote is: don't do computed setters for arrays. (Kind of makes sense: if it's mutated by calling a method on the same object, it's not a setter. duh!)

So the right way :tm: to do this is probably to just have a `wikibaseItems() { /* getter code */ }` computed property (no `get`/`set` pair) and instead of using a setter we write to the store through separate `addItem`, `deleteItem`, `editItem` methods for which we can either create separate mutations or have them call the `setItemMapping` as we do now in the setter.